src/pipelines/epidemiology/config.yaml

```diff
@@ -421,8 +417,6 @@ sources:
 
   - name: pipelines.epidemiology.gb_authority.Covid19UkL3DataSource
     test:
-      # TODO(owahltinez): skip test since it's very flakey
```

Comment:
Unfortunately these tests remain flakey - I notice the API goes down after 5pm on Monday pretty regularly for instance, and sometimes the L3 data source simply times out (presumably if the answer is not in their cache yet).

Reply:
Got it, I'll keep the skip test then